packages/cardano-services/src/TxWorker/loadTxWorker.ts

```diff
@@ -12,12 +12,19 @@ export interface TxWorkerArgs {
 }
 
 export const loadTxWorker = async (args: TxWorkerArgs) => {
-  const { loggerMinSeverity, ogmiosUrl, rabbitmqUrl } = args.options;
-  const txSubmitProvider = ogmiosTxSubmitProvider(urlToConnectionConfig(ogmiosUrl));
+  const { loggerMinSeverity, serviceDiscoveryBackoffFactor, serviceDiscoveryBackoffTimeout, cacheTtl } = args.options;
   const logger = createLogger({ level: loggerMinSeverity, name: 'tx-worker' });
+  const cache = new InMemoryCache(cacheTtl);
+  const dnsResolver = createDnsResolver(
```

Comment:
The entire file LGTM. I simply have not clear how this can work when the related options are not provided from command line... but looking at the tests, it is clear that or it works or tests will fail!

Reply:
Ok, now it's clear, thank you